source/PyMaterialX/PyMaterialXGenGlsl/PyGlslShaderGenerator.cpp

```diff
@@ -48,3 +49,15 @@ void bindPyEsslShaderGenerator(py::module& mod)
         .def("getTarget", &mx::EsslShaderGenerator::getTarget)
         .def("getVersion", &mx::EsslShaderGenerator::getVersion);
 }
+
+// Glsl Vulkan shader generator bindings
+
+void bindPyVkShaderGenerator(py::module& mod)
+{
+    py::class_<mx::VkShaderGenerator, mx::HwShaderGenerator, mx::VkShaderGeneratorPtr>(mod, "VkShaderGenerator")
```

Comment:
Assuming the second argument in this list represents the base class, shouldn't it be `mx::GlslShaderGenerator` rather than `mx::HwShaderGenerator`?  (It looks like we should apply the same change to the bindings for `mx::EsslShaderGenerator` above as well!)